src/Application/Models/Asset.js

```diff
@@ -1,13 +1,15 @@
 import { assetDefaults } from './../../util/constants';
 
 class Asset {
-  constructor(name, price, multiplier, owned, unlocked) {
+  constructor(name, price, multiplier, owned, unlocked, investment, revenue) {
     this._name = name;
     this._price = price;
     this._multiplier = multiplier;
     this._owned = owned;
     this._unlocked = unlocked;
     this._defaults = assetDefaults[name];
+    this._investment = investment;
```

Comment:
I care a lot about the naming of things, since it's the primary motor for self documenting code.  Everytime you're about to name something, consider how much contextual understanding you're bringing to it, how much you're assuming other people (or yourself in the future, perhaps more importantly) know about the code as a whole, and try to reduce those assumptions to 0.  Surely, it'll never be possible to understand a variable just from the name with 0 contextual understanding, but the closer you can get it the better.